jenkins_jobs/modules/parameters.py

```diff
@@ -1171,4 +1387,19 @@ def gen_xml(self, xml_parent, data):
             if pdefs is None:
                 pdefs = XML.SubElement(pdefp, "parameterDefinitions")
             for param in parameters:
+                # Pass job name to the uno-choice plugin
+                if next(iter(param)) == "active-choices":
+                    param["active-choices"]["_project-name"] = data.get(
```

Comment:
To avoid duplicating stuff I propose to rework it to:

    param_type = next(iter(param))
    if param_type in ("active-choices", "dynamic-reference"):
        param[param_type][...] = ...